types/fs-ext/index.d.ts

```diff
@@ -1,101 +1,185 @@
-// Type definitions for fs-ext
+// Type definitions for fs-ext 2.0.0
 // Project: https://github.com/baudehlo/node-fs-ext
 // Definitions by: Oguzhan Ergin <https://github.com/OguzhanE>
+//                 Konstantin Ikonnikov <https://github.com/ikokostya>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-///<reference types="node"/>
-
-
-export * from "fs";
```

Comment:
This is not needed anymore, because since v2.0.0 module `fs-ext` doesn't override Node.js core `fs` module.

Reply:
Perfect!